mutational-signatures/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu
+RUN apt update
+# Needed to supress r-base region messages
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt install -y r-base r-cran-devtools libgsl-dev python3-pip wget
+RUN Rscript -e "devtools::install_github('mskcc/temposig')"
```

Comment:
prolly good if this is some specific version - and have some way to capture on e.g. datahub what specific version was run to generate the signature data